libs/blocks/caas-marquee/caas-marquee.css

```diff
@@ -798,3 +798,782 @@
     max-width: 800px;
   }
 }
+
+/* marquee temp support. */
```

Comment:
Is this temporary?

Reply:
This line of css was over copied from the reference implementation. I am not sure the context of that css comment.